Update Powershell command

options opened the pull request:
Get-AzureRMVMImagePublisher --> Get-AzureRmVMImagePublisher 
Get-AzureRMVMImageOffer --> Get-AzureRmVMImageOffer

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (luisca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

options committed:
Update Powershell command

Get-AzureRMVMImagePublisher --> Get-AzureRmVMImagePublisher 
Get-AzureRMVMImageOffer --> Get-AzureRmVMImageOffer
